issue_template.md

```diff
@@ -0,0 +1,18 @@
+Please include the following information before submitting your issue
+Any information not included may be requested when diagnosing the issue
+
+Description of the Issue
+------------------------
```

Comment:
Instead of the dashes user a bold styling for the headline? Using two asterisks for example?
``**Description of the Issue**``

Reply:
@baka0815 They were meant to be headings so that they are not replaced with content.